src/models/Expense.model.js

```diff
@@ -1,9 +1,43 @@
 'use strict';
 
 const { sequelize } = require('../db.js');
+const { DataTypes } = require('sequelize');
 
 const Expense = sequelize.define(
-  // your code goes here
+  'Expense',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+    },
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    spentAt: {
+      type: DataTypes.TEXT,
```

Comment:
The 'spentAt' field is currently using DataTypes.TEXT. It might be more appropriate to use DataTypes.DATE or DataTypes.DATEONLY if this field is intended to store date values.